src/components/MoneyRequestHeader.js

```diff
@@ -82,16 +83,21 @@ function MoneyRequestHeader({session, parentReport, report, parentReportAction,
 
     const isScanning = TransactionUtils.hasReceipt(transaction) && TransactionUtils.isReceiptBeingScanned(transaction);
     const isPending = TransactionUtils.isExpensifyCardTransaction(transaction) && TransactionUtils.isPending(transaction);
-
     const canModifyRequest = isActionOwner && !isSettled && !isApproved && !ReportActionsUtils.isDeletedAction(parentReportAction);
+    let canDeleteRequest = canModifyRequest;
+
+    if (ReportUtils.isPaidGroupPolicyExpenseReport(moneyRequestReport)) {
+        // If it's a paid policy expense report, only allow deleting the request if it's not submitted or the user is the policy admin
+        canDeleteRequest = canDeleteRequest && (ReportUtils.isDraftExpenseReport(moneyRequestReport) || PolicyUtils.isPolicyAdmin(policy));
+    }
```

Comment:
```suggestion

    // If it's a paid policy expense report, only allow deleting the request if it's not submitted or the user is the policy admin
    const canDeleteRequest =
        canModifyRequest &&
        (ReportUtils.isPaidGroupPolicyExpenseReport(moneyRequestReport) ? ReportUtils.isDraftExpenseReport(moneyRequestReport) || PolicyUtils.isPolicyAdmin(policy) : true);
```

NAB. But I think using const is better for this case. It's easier to know where canDeleteRequest value is coming from

Reply:
Hmm, I feel the current code is easier to read